users/tests/test_unit_customuser_model.py

```diff
@@ -17,13 +17,15 @@
 class TestCustomUserModel(TestCase):
     def test_create_user_method_should_raise_custom_validation_error_when_email_is_none(self):
         with self.assertRaises(ValidationError) as custom_exception:
-            CustomUser.objects._create_user(None, "testuserpasswd", False, False, CustomUser.UserType.EMPLOYEE.name)
+            CustomUser.objects._create_user(
+                None, "testuserpasswd", False, False, CustomUser.UserType.EMPLOYEE.name, True
+            )
```

Comment:
Can you please use here positional parameters? :) Without them every time whoever look at this method needs to go inside to look to what parameters there values corresponds.

Reply:
Sure, done.